go.mod

```diff
@@ -86,3 +87,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace golang.org/x/net v0.23.0 => golang.org/x/net v0.39.0
```

Comment:
this version of `net` has some vulnerabilities so i upgraded it but still added the replace to catch indirect dependencies